src/w3d/lib/rawfileclass.cpp

```diff
@@ -182,10 +182,10 @@ bool RawFileClass::Open(int rights)
 
     switch (rights) {
         case FM_READ:
-            m_handle = open(m_filename, O_RDONLY, S_IREAD);
+            m_handle = open(m_filename, O_RDONLY, S_IREAD | S_IWRITE);
```

Comment:
If I am not mistaken this means no 2 reads can access this file at the same time. This does not seem right.

Reply:
No, that should have no impact. It's only about the permission flags that the files get created with. We're still opening the file with `O_RDONLY` which is the readmode

Reply:
Ok. Is this how original Thyme does it? If not, we need BUGFIX comment explaining the fix.

Reply:
It's a bug that we introduced. This codepath does not exist in the original